packages/primitives/paragraph/src/types.ts

```diff
@@ -2,9 +2,12 @@ import { ReactNode } from 'react'
 import { TThemeableText } from '@themeables/text'
 import { TThemeableSpacer } from '@themeables/spacer'
 
+export type TAlign = 'start' | 'center' | 'end'
```

Comment:
Can we pull these two types from the `TThemeableTextAlign` type?

Reply:
Sure 👍